vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/AndroidTestParser.kt

```diff
@@ -28,10 +28,6 @@ class AndroidTestParser : TestParser {
             val className = packageAndClassName.substring(lastDotIndex + 1 until packageAndClassName.length)
 
             Test(packageName, className, methodName, annotations)
-        }.also {
-            if (it.isEmpty()) {
```

Comment:
Why this code was removed?

Reply:
It was moved to Marathon.kt into the core